index.css

```diff
@@ -371,3 +384,253 @@ textarea::placeholder {
   margin-bottom: 60px;
   background-color: white;
 }
+
+.nav-links {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  body {
+    background: #e5e5e5;
+  }
+  header { background: white; }
+
+  .bar-line {
+    display: none;
+  }
+
+  .nav-links {
+    display: block;
+    margin-right: 20vw;
+  }
+
+  .nav-links > ul {
+    display: flex;
+    list-style: none;
+  }
+
+  .nav-links > ul > li > a {
+    text-decoration: none;
+    color: #344563;
+  }
+
+  .card-divider > ul > li {
+    background-color: #ebebff;
+    padding: 5px 5px 4px 4px;
+    font-size: 18px;
+    color: #6070ff;
+    border-radius: 5px;
+  }
+
+  .nav-links > ul > li:nth-child(2) {
+    margin-left: 8px;
+  }
+
+  .nav-links > ul > li:nth-child(3) {
+    margin-left: 8px;
+  }
+
+  .header-nav {
+    width: 100vw;
+    padding-left: 50px;
+    overflow: hidden;
+  }
+
+  .header-section {
+    overflow: hidden;
+  }
+
+  .header-herosection {
+    background-image: url("/images/Header\ bg.svg");
+    background-size: 100% 100%;
+    width: 100vw;
+    height: 88vh;
+    overflow-x: hidden;
+    border-bottom-left-radius: 100px;
+  }
+
+  .header-herosection > h1 {
+    padding-right: 30vh;
+    padding-left: 18vw;
+    line-height: 38px;
+    margin-top: 20vh;
+  }
+
+  .header-herosection > p {
+    padding-left: 18vw;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    padding-right: 20vw;
+  }
+
+  .header-herosection > h3 {
+    padding-left: 18vw;
+    margin-block-start: 0;
+    margin-block-end: 0;
+  }
+
+  .social-media {
+    padding-left: 18vw;
+    margin-block-start: 0;
+    margin-block-end: 0;
+  }
+
+  .grid-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .grid-item {
+    display: flex;
+    flex-direction: row;
+    width: 90%;
+    height: 300px;
+    justify-content: space-between;
+    background: white;
+  }
```

Comment:
- [ ] Please make sure the changes you introduce are aligned with the Figma design as I've noticed you're project images are stretching too much. see below 👇
  ![image](https://user-images.githubusercontent.com/104968236/236446251-42795a7c-b6f1-48a3-9c3f-31fad67d8a46.png)
- [ ] So in order to fix this issue you can start by removing the fixed heights you've set: 
    - [ ] 1st remove `height: 300px;` from the `.grid-item` class 
    - [ ] 2nd remove `height: 280px;` from the `.image-wrapper > img` selector at line `498`. 
    - [ ] Finally, you can make some other refinements in accordance 😉